docs/algorithms.rst

```diff
@@ -39,19 +39,20 @@ Specifying an Algorithm
 You can specify which algorithm you would like to use to sign the JWT
 by using the `algorithm` parameter:
 
-.. code-block:: python
+.. code-block:: pycon
```

Comment:
is it pycon or python?

Reply:
pycon

See here for a complete list of Pygment lexers, which are used by Sphinx: https://pygments.org/docs/lexers/#pygments.lexers.python.PythonConsoleLexer

This code block represents an interactive Python console session. It does not represent a Python script, therefore, pycon is the better choice for code highlighting which understands the `>>> ` prompt.

Same applies to the other review comment.

You can see the same thing in other projects, e.g. Django:
https://github.com/django/django/blob/bb64b99b78a579cb2f6178011a4cf9366e634438/docs/intro/tutorial05.txt#L149-L158

Reply:
didn't know that. thanks for explaining :D